File: NetBeans_IDE/JsonPL/src/com/middlemind/JsonPL/ProcessUrlCallRunner.java
package com.middlemind.JsonPL;

/**
 *
 * @author brusc
 */
public class ProcessUrlCallRunner implements Runnable {
    private final JsonPlState main;
    private final String url;
    
    public ProcessUrlCallRunner(JsonPlState Main, String Url) {
        main = Main;
        url = Url;
    }
    
    @Override
    public void run() {
        main.processUrlCallExec(url, true);
    }    
}

File: VisualStudio_IDE/JsonPL/JsonPL/com/middlemind/JsonPL/ProcessUrlCallRunner.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace com.middlemind.JsonPL
{
    public class ProcessUrlCallRunner
    {
        private JsonPlState main;
        private string url;

        public ProcessUrlCallRunner(JsonPlState Main, string Url)
        {
            main = Main;
            url = Url;
        }

        public void run()
        {
            main.processUrlCallExec(url, true);
        }
    }
}
